recipe/meta.yaml

```diff
@@ -27,11 +27,12 @@ requirements:
     - bw_processing >=0.9.5
     - json-logging
     - matrix_utils >=0.4.3
-    - numpy <2
     - pandas
     - pydantic
     - scipy
     - stats_arrays
+    - numpy <2  # [not (osx and arm64)]
+    - numpy <=1.24  # [osx and arm64]
```

Comment:
If it breaks on NumPy 1.26, why do you also exclude NumPy 1.25?

Reply:
Pure naked discrimination :)

The `Tester` was removed in 1.25: https://numpy.org/doc/stable/release/1.25.0-notes.html